Jetsurvey/gradle/libs.versions.toml.ignore

```diff
@@ -0,0 +1,66 @@
+[versions]
+androidGradlePlugin = "7.2.1"
```

Comment:
Was this file meant to be committed? .ignore?

Reply:
Oops, no. Needs to be deleted